packages/drivers/routerlicious-driver/src/index.ts

```diff
@@ -7,6 +7,9 @@
 export { DefaultTokenProvider } from "./defaultTokenProvider";
 export { ITokenProvider, ITokenResponse, ITokenService } from "./tokens";
 
+// Errors
+export { R11sErrorType } from "./errorUtils";
```

Comment:
Since it looks like this wasn't exported previously (and so there is little danger of breaking anyone), can we rename this to spell out "Routerlicious"? I don't think the abbreviated `R11s` term is good from an accessibility perspective, so we shouldn't use that terminology publicly (or even privately, IMO).